pkg/utils/k8sutil.go

```
@@ -847,3 +658,278 @@ func DeleteIngress(client kubernetes.Interface, name, ns string) error {
 	}
 	return nil
 }
+
+func splitHandler(handler string) (string, string, error) {
+	str := strings.Split(handler, ".")
+	if len(str) != 2 {
+		return "", "", errors.New("Failed: incorrect handler format. It should be module_name.handler_name")
+	}
+
+	return str[0], str[1], nil
+}
+
+func ensureFuncConfigMap(client kubernetes.Interface, funcObj *spec.Function, or []metav1.OwnerReference) error {
+	modName, _, err := splitHandler(funcObj.Spec.Handler)
+	if err != nil {
+		return err
+	}
+
+	fileName := modName
+	_, depName, fileName, err := GetFunctionData(funcObj.Spec.Runtime, funcObj.Spec.Type, modName)
+	if err != nil {
+		return err
+	}
+
+	configMap := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            funcObj.Metadata.Name,
+			Labels:          funcObj.Metadata.Labels,
+			OwnerReferences: or,
+		},
+		Data: map[string]string{
+			"handler": funcObj.Spec.Handler,
+			fileName:  funcObj.Spec.Function,
+			depName:   funcObj.Spec.Deps,
+		},
+	}
+
+	_, err = client.Core().ConfigMaps(funcObj.Metadata.Namespace).Create(configMap)
+	if err != nil && k8sErrors.IsAlreadyExists(err) {
+		data, _ := json.Marshal(configMap)
```

should not ignore error here